core/src/components/searchbar/searchbar.tsx

```diff
@@ -288,7 +289,7 @@ export class Searchbar implements ComponentInterface {
       // Create a dummy span to get the placeholder width
       const doc = this.doc;
       const tempSpan = doc.createElement('span');
-      tempSpan.innerHTML = this.placeholder;
+      tempSpan.innerHTML = sanitizeDOMString(this.placeholder) || '';
```

Comment:
So this one I'm not sure if we even need since it looks like it's a temporary element, but I figured it couldn't hurt to add it.